src/data/seed.js

```diff
@@ -4,10 +4,16 @@ import {
   down as flushUser,
 } from './seeders/user';
 
+import {
+  up as seedSubmission,
+  down as flushSubmission,
+} from './seeders/submission';
+
 const up = async () => {
   console.log('Starting to seed');
   try {
     await seedUser();
+    await seedSubmission();
```

Comment:
can these be done in parallel? Something like 
`await Promise.all[seedUser(), seedSubmission*()]`